packages/sui-hoc/src/withOpenToggle.js

```diff
@@ -5,7 +5,7 @@ export default BaseComponent => {
 
   return class WithOpenToggle extends Component {
     state = {
-      isOpen: false
+      isOpen: !!this.props.isOpen // eslint-disable-line
```

Comment:
🍆 do you still need to disable this line on eslint?

Reply:
🤪  👉🏼   https://github.com/SUI-Components/sui/pull/1002#discussion_r531574376

Reply:
sorry, hadn't seen the resolved convo! ;)